spec/prog/vnet/load_balancer_nexus_spec.rb

```diff
@@ -148,10 +148,22 @@
       expect { nx.wait_cert_broadcast }.to nap(1)
     end
 
-    it "hops to wait if all children are done" do
+    it "hops to wait if all children are done and no certs to remove" do
       expect(nx).to receive(:reap)
       expect { nx.wait_cert_broadcast }.to hop("wait")
     end
+
+    it "removes certs if all children are done and there are certs to remove" do
```

Comment:
I recommend a test where the certificate is associated with multiple load balancers, to ensure it is only removed from the expected load balancer.

Reply:
That's not really a valid scenario, a cert can only be associated with a single load balancer.